### ISSUE TITLE
Allow the user to add custom coins

### ISSUE THREAD

**Issue by Senyoret1** [skycoin/skycoin-web#482]
Currently the wallet only allows the user to select among the coins that have been added in the code. It should be possible to connect the wallet to fiber coins that have not yet been added to the code, allowing the user to manually add the information about the coin.

@gz-c , I suppose this is part of what you wanted to be added in https://github.com/skycoin/skycoin-web/issues/464 , apart from modifying the URL of the coins that are already available to the user, I am right?

**Comment by gz-c** [comment]
Yes, the user would be able to add their own coin/domain pair.  Once loaded, the wallet will query some url on the domain for configuration data on how to interact with the coin.  Things like coin hour burn, name, possibly color scheme will be exposed.

**Comment by olemis** [comment]
@gz-c I don't think  `/api/v1/blockchain/metadata` provides enough info for that . Shall we create a new issue for `skycoin/skycoin` ?

**Comment by gz-c** [comment]
@olemis yes create a new issue if there isn't one already
